api/src/main/java/io/opencensus/trace/BlankSpan.java

```diff
@@ -34,8 +34,13 @@ private BlankSpan() {
     super(SpanContext.INVALID, null);
   }
 
+  /** No-op implementation of the {@link Span#putAttribute(String, AttributeValue)} method. */
+  @Override
+  public void putAttribute(String key, AttributeValue value) {}
+
   /** No-op implementation of the {@link Span#addAttributes(Map)} method. */
   @Override
+  @SuppressWarnings("deprecation")
   public void addAttributes(Map<String, AttributeValue> attributes) {}
```

Comment:
This method should have a Deprecation annotation instead of SuppressWarnings, if you also want to remove it from BlankSpan.